radicle-cli/src/terminal/cob.rs

```diff
@@ -0,0 +1,15 @@
+use std::str::FromStr;
+
+use super::*;
+use radicle::cob::patch;
+
+use anyhow::anyhow;
+
+pub fn parse_patch_id(val: OsString) -> Result<patch::PatchId, anyhow::Error> {
+    let val = val
+        .to_str()
+        .ok_or_else(|| anyhow!("patch id specified is not UTF-8"))?;
+    let patch_id =
+        patch::PatchId::from_str(val).map_err(|_| anyhow!("invalid patch id '{}'", val))?;
+    Ok(patch_id)
+}
```

Comment:
I think that `PatchId` is just a `type` alias for `ObjectId`. So this could be used for any `cob`, in theory. I wonder if it would make sense to just call this `parse_identifier` and use it for `PatchId`, `IssueId`, etc.

Reply:
You're right about the type alias, however the error message is different.

Reply:
Yeah I think we'll want to do something like `cob::parse_id(&id, "patch")` and `cob::parse_id(&id, "issue")`